.env.example

```
@@ -135,6 +135,19 @@ DISCORD_ALLOWED_USER_IDS=
 # Get your user ID by messaging @userinfobot on Telegram
 TELEGRAM_ALLOWED_USER_IDS=
 
+# Zulip Bot (community adapter)
+# Create a bot in Zulip: Settings > Personal settings > Bots > Add a new bot
+ZULIP_URL=                 # Zulip realm URL, e.g. https://your-org.zulipchat.com
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)